### ISSUE TITLE
GlobalAveragePooling2D error 'Tensor' object has no attribute 'lower'

### ISSUE THREAD

**Issue by hadaev8** [keras-team/keras#9614]
Same for GlobalMaxPooling2D

keras from master, tensorflow from tf-nightly-gpu

```def disc():
    def conv(i, deph, size, std, usebn = True):
        i = Conv2D(filters = deph, kernel_size = size, strides = 1, padding = 'same')(i)
        if usebn:
            i = bn(i)
        i = relu(i)
        i = MaxPooling2D(std)(i)
        return i

    inp = Input(shape = d_shape)
    i = inp

    i = conv(i, depth, size, 2, False)    
    i = conv(i, depth, size, 2)
    i = conv(i, depth, size, 2)
    i = conv(i, depth, size, 2)
    
    i = GlobalMaxPooling2D(i)

    i = Dense(len(labels), activation = 'softmax')(i)

    m = Model(inp, i)

**Comment by Dref360** [comment]
GlobalMaxPooling2D()(i)

**Comment by hadaev8** [comment]
Damm, pretty obvious, thank you.
